### ISSUE TITLE
NMF removed from CRAN

### ISSUE THREAD

**Issue by rekado** [renozao/NMF#106]
Visiting the [NMF page on CRAN](https://cran.r-project.org/web/packages/NMF/index.html), I get this message:

> Package ‘NMF’ was removed from the CRAN repository.
> 
> Formerly available versions can be obtained from the archive.
> 
> Orphaned on 2018-02-23 as maintainer email addressed bounced after months of non-response.. Archived on 2018-03-02.

Will NMF return to CRAN at some point?

**Comment by renozao** [comment]
Yes. I changed affiliation and I need to reflect this. Will try update it
soon.

**Comment by Dybian** [comment]
Hi,
A new release appeared on CRAN but it seems quite old (0.21.0) compared to versions available here.
It could be useful to release a more recent version which fix https://bugs.debian.org/879248.
